### PR TITLE
add fuse and bump integreatly operator

### PR DESCRIPTION
**Pull request by maleck13**
@philbrookes TIL I think I have done this right

### CODE REVIEW

File: integreatly/1.1.0/integreatly-operator.1.1.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,243 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: integreatlyoperator.1.0.1
```
Comment (philbrookes) [inline integreatly/1.1.0/integreatly-operator.1.1.0.clusterserviceversion.yaml]: this needs to change too

File: integreatly/1.1.0/integreatly-operator.1.1.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+    containerImage: quay.io/integreatly/integreatly-operator:1.1.0
+    createdAt: 2019-04-30 12:39:00
+    support: Integreatly
+    description: Greatly integrating integreatly greatness
+
+spec:
+  displayName: Integreatly
+  description: Integreatly installation and reconciliation operator
+  keywords: ['integreatly']
+  version: 1.0.1
```
Comment (philbrookes) [inline integreatly/1.1.0/integreatly-operator.1.1.0.clusterserviceversion.yaml]: and here

File: integreatly/1.1.0/integreatly-operator.1.1.0.clusterserviceversion.yaml [hunk 3]
```diff
@@ -0,0 +1,243 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: integreatlyoperator.1.0.1
+  namespace: placeholder
+  annotations:
+    alm-examples: "{}"
+    categories: Integreating greatly
+    certified: 'false'
+    containerImage: quay.io/integreatly/integreatly-operator:1.1.0
```
Comment (philbrookes) [inline integreatly/1.1.0/integreatly-operator.1.1.0.clusterserviceversion.yaml]: has this image been pushed to this repo?

File: integreatly/1.1.0/integreatly-operator.1.1.0.clusterserviceversion.yaml (hunk 1 above)
Comment (maleck13) [reply integreatly/1.1.0/integreatly-operator.1.1.0.clusterserviceversion.yaml]: Yeah I had to change it back due to an error from olm about 2 csvs. Doing this got it working

File: integreatly/1.1.0/integreatly-operator.1.1.0.clusterserviceversion.yaml (hunk 3 above)
Comment (maleck13) [reply integreatly/1.1.0/integreatly-operator.1.1.0.clusterserviceversion.yaml]: yep

File: integreatly/1.1.0/integreatly-operator.1.1.0.clusterserviceversion.yaml (hunk 1 above)
Comment (maleck13) [reply integreatly/1.1.0/integreatly-operator.1.1.0.clusterserviceversion.yaml]: moved back to 1.0.1 but updated the image

File: Makefile [hunk 4]
```diff
@@ -4,8 +4,9 @@ INTEGREATLY_VERSION=1.0.1
 AMQSTREAMS_VERSION=1.1.0
 RHSSO_VERSION=1.6.1
 CODEREADY_VERSION=1.2.2
+FUSE_VERSION=1.7.0
 
-push/all: push/integreatly push/amqstreams push/3scale push/rhsso push/codeready
+push/all: push/integreatly push/amqstreams push/3scale push/fuse push/codeready
```
Comment (philbrookes) [inline Makefile]: why have you removed rhsso?

Comment (maleck13) [reply Makefile]: woops will change

Comment (maleck13) [reply Makefile]: fixed

**Pull request merged**